### DISCUSSION TITLE
Unable to deploy node unblocker through Heroku

### DISCUSSION THREAD

**Discussion by IDK452** [nfriedly/nodeunblocker.com#38] [General]
Whenever I click the button that takes me directly to Heroku and hit deploy app it always stops at the build app point and says this "There was an issue building your app. This can mean your app.json's project is not a valid Heroku application. Please ensure your app is deployable to Heroku and try again." Now tbh I don't know any code what so ever so hopefully you could explain this to me whether it is a problem on my end or yours.

**Comment by kiprasvitas** [comment]
Unfortunately, Heroku has stopped deploying all proxy-related repositories due to updates in their terms of service. I think a major contributor to this issue, however, is simply that the application does not have the updated versions of gatling. I am working on a fix.

**Comment by IDK452** [comment]
That makes sense thanks for your reply also wanted to ask if you know any other sites that I would be able to run this on if you do please let me know

**Comment by begumAnthony** [comment]
where else can I deploy this code too if it hasn't expired yet and when recommending a new app could you please give me a step by step tutorial

**Comment by nfriedly** [comment]
Yeah, a while back, when I used to keep a copy of this site online, heroku shut down my instance because of a complaint. The complaint turned out to be bogus, but they told me that they were going to change their TOS to disallow proxies, and so they weren't going to re-enable mine even. I stopped using heroku after that.
